samples/GraphQL.DataLoader.Sample.Default/DealershipDbContext.cs

```diff
@@ -0,0 +1,90 @@
+using GraphQL.DataLoader.Di.Sample.Types;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace GraphQL.DataLoader.Di.Sample;
+
+public class DealershipDbContext : DbContext
+{
+    public DbSet<Car> Cars => Set<Car>();
+    public DbSet<Salesperson> Salespeople => Set<Salesperson>();
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder
+            .UseSqlite(
+                new SqliteConnectionStringBuilder()
+                {
+                    Mode = SqliteOpenMode.ReadWriteCreate,
+                    BrowsableConnectionString = true,
+                    DataSource = "my.db"
+                }
+                .ToString()
+            )
+            .UseSeeding((ctx, _) =>
+            {
+                if (ctx.Set<Salesperson>().Any())
+                    return;
+                ctx.Set<Salesperson>().AddRange(_seedSalespeople);
+                ctx.SaveChanges();
+            })
+            .UseAsyncSeeding(async (ctx, _, cancellationToken) =>
+            {
+                if (await ctx
+                        .Set<Salesperson>()
+                        .AnyAsync(cancellationToken)
+                        .ConfigureAwait(false))
```

Comment:
As a general rule, all calls to ConfigureAwait within application code should be removed.  Not sure if one of the formatting rules needs to be changed for these sample projects.  `ConfigureAwait(false)` should be used for all library calls.

For reference, ConfigureAwait is used to return calls back to the original synchronization context.  However, there is none within ASP.NET Core, so `ConfigureAwait(false)` is a no-op.  This isn't exactly the reason for the above guidelines, but it does mean that it is unnecessary in any ASP.NET Core application, which these samples are.  So it will be cleaner if we remove the unnecessary code from the sample projects.

Here's one of many docs on the use of ConfigureAwait:
- https://devblogs.microsoft.com/dotnet/configureawait-faq/

Reply:
ah ok thx. i actually didnt realize there is no Syncronization Context in Asp.NetCore. In my Last Job adding this everywhere (in App Level Code of Asp.NetCore WebApis) was actually enforced with warnings. So at this point i thought it good practice.
I'll remove it from the sample Projects then.

Reply:
ye im getting Build Errors about removing the Configure await now @Shane32 could you help me out where to exclude the 2 new Sample Projects from this Rule?

Reply:
i added it to directory.build.targets -- see commit.